performance/tests/test_eqprs_api.py

```diff
@@ -0,0 +1,35 @@
+#from django.test import TestCase
+import json
+from django.urls import reverse
+from rest_framework.test import APITestCase,APIRequestFactory
+from rest_framework import status
+from performance.views import IndicatorReadOnlyModelViewSet
+from performance.serializer import IndicatorSerializer
+from performance.models import Indicator
+# Create your tests here.
+
+
+class indicator_API_Test(APITestCase):
+    list_url = reverse("indicator-list")
+    fixture = ['test_api.json']
+    
+    def test_api(self):
+        response = self.client.get(self.list_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
I think a basic assertion that the API contains an expected field and value would be good. We don't need to test this comprehensively since we trust that django-rest-framework's model serializer is well-tested, but something rough like that there are as many objects in the response as expected, and that the indicator name is as we expect, would be good to detect if someone goes and totally breaks the django-rest-framework configuration

Reply:
See how to access the response body at https://docs.djangoproject.com/en/2.2/topics/testing/tools/